src/wms/src/main/java/org/geoserver/wms/legendgraphic/BufferedImageLegendGraphicBuilder.java

```diff
@@ -172,7 +171,7 @@ public BufferedImage buildLegendGraphic(GetLegendGraphicRequest request)
             } else if (buildRasterLegend) {
                 final RasterLayerLegendHelper rasterLegendHelper =
                         new RasterLayerLegendHelper(request, gt2Style, ruleName);
-                final BufferedImage image = rasterLegendHelper.getLegend();
+                BufferedImage image = rasterLegendHelper.getLegend();
```

Comment:
Why is "final" gone here? Was expecting to see it being reassigned, but cannot find where.

Reply:
fixed